README.md

```diff
@@ -696,6 +696,7 @@ This list results from Pull Requests, reviews, ideas, and work done by 1600+ peo
   * [Aserto](https://www.aserto.com) - Fine-grained authorization as a service for applications and APIs. Free up to 1000 MAUs and 100 authorizer instances.
   * [asgardeo.io](https://wso2.com/asgardeo) - Seamless Integration of SSO, MFA, passwordless auth and more. Includes SDKs for frontend and backend apps. Free up to 1000 MAUs and five identity providers.
   * [Auth0](https://auth0.com/) — Hosted SSO. Up to 7000 active users and two social identity providers.
+  * [Auth.js (NextAuth.js)](https://authjs.dev/) — Runtime agnostic library based on standard Web APIs that integrates deeply with multiple modern JavaScript frameworks.
```

Comment:
Please describe the SaaS features this provide. 

This is a list for SaaS offerings.